src/charm.py

```diff
@@ -910,8 +910,11 @@ def _refresh_firewall(self):
                 FirewallEntry.decode(entry.strip())
                 for entry in firewall_denylist_config.split(",")
             ]
+        allowlist = [
+            FirewallEntry.decode(str(entry.host)) for entry in self._state.ssh_debug_infos
```

Comment:
minor point, information cannot be plural in english

Reply:
Renamed, thank you!